src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -3185,6 +3189,160 @@ default Task<List<Member>> retrieveMembersByIds(boolean includePresence, @Nonnul
+        return getAutoModerationRuleCache().getElementById(id);
+    }
+
+    /**
+     * Gets a list of all auto moderation rules from the cache.
+     *
+     * @return A list of all auto moderation rules, or an empty list if none exist.
+     */
+    @Nullable
+    default List<AutoModerationRule> getAutoModerationRules() {
+        return getAutoModerationRuleCache().asList();
+    }
+
+    /**
+     * Retrieves a list of the auto moderation rules which are configured for this guild.
+     *
+     * @param id, the id of the rule you want to retrieve.
+     * @return A list of auto moderation rule objects for the given guild.
+     */
+    @Nonnull
+    RestAction<AutoModerationRule> retrieveAutoModerationRule(String id);
+
+    /**
+     * Retrieves a list of the auto moderation rules which are configured for this guild.
+     *
+     * @param id, the id of the rule you want to retrieve.
+     * @return A list of auto moderation rule objects for the given guild.
+     */
+    default RestAction<AutoModerationRule> retrieveAutoModerationRule(long id) {
+        return retrieveAutoModerationRule(Long.toUnsignedString(id));
+    }
+
+    /**
+     * Retrieves a list of the auto moderation rules which are configured for this guild.
+     *
+     * @return A list of auto moderation rule objects for the given guild.
+     */
+    @Nonnull
+    RestAction<List<AutoModerationRule>> retrieveAutoModerationRules();
+
+    /**
+     * This can be used to create a {@link AutoModerationRule rule} for this guild.
+     *
+     * @param name, the name of the rule you want to create.
+     * @param eventType, the event which will cause the auto moderation system to check for a specific trigger.
+     * @param triggerType, the type of trigger that can cause an auto moderation rule to be executed.
+     * @param triggerMetadata, This is additional data used whether a rule should be executed or not.
+     * @param actions, the action that will be performed when the rule is triggered.
+     * @param enabled, whether the rule is enabled or not.
+     * @param exemptRoles, the roles that will be exempt from the rule.
+     * @param exemptChannels, the channels that will be exempt from the rule.
+     *
+     * @return The new {@link AutoModerationRule rule} that was created.
+     */
+    @Nonnull
+    AuditableRestAction<AutoModerationRule> createAutoModerationRule(@Nonnull String name, @Nonnull EventType eventType, @Nonnull TriggerType triggerType, @Nullable TriggerMetadata triggerMetadata, @Nonnull List<AutoModerationAction> actions, boolean enabled, @Nullable List<Role> exemptRoles, @Nullable List<Channel> exemptChannels);
+
+    /**
+     * This can be used to modify an existing {@link AutoModerationRule rule's} name which is in this guild.
+     *
+     * @param rule, the rule which you want to modify.
+     * @param name, modify the name of the rule.
+     *
+     * @return an auto moderation rule.
```

Comment:
```suggestion
     * @return {@link net.dv8tion.jda.core.requests.restaction.AuditableRestAction AuditableRestAction} - Type: {@link AutoModerationRule}
     *         <br>The modified AutoModerationRule after the rule name was modified to the specified name
```

Also, please follow JDA's structure guide for Javadoc